tests/test_import.py

```diff
@@ -41,9 +41,9 @@ def test_import_prepHistFactory():
     assert pdf.expected_actualdata(
         pdf.config.suggested_init()).tolist() == [120.0, 110.0]
 
-    assert pdf.config.auxdata_order == ['syst1', 'staterror_channel1', 'syst2', 'syst3']
+    assert pdf.config.auxdata_order == ['staterror_channel1', 'syst1', 'syst2', 'syst3']
```

Comment:
could also change this to `sorted(['syst1', 'staterror_channel1', 'syst2', 'syst3'])` makes the diff smaller and intent is clearer